sql/geometry_entity_expiry_date.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE civigeometry_geometry_entity ADD COLUMN `expiry_date` timestamp NULL  DEFAULT NULL COMMENT 'When Should this geometry entity relationship expire';
+ALTER TABLE civigeometry_geometry_entity ADD INDEX `index_expiry_date`(`expiry_date`);
```

Comment:
@johntwyman @unfinishedteleporter given we maybe running a query on this table where the only thing in the Where is an expiry date figured it was useful to put an index on that column by its self

Reply:
Agreed.